week0001/fizz_buzz_haond1.rb

```diff
@@ -0,0 +1,8 @@
+FIZZ = [1,2,2]
+def fizz_buzz(n)
+  (n/5).times{|i|
+    FIZZ[i%3].times{ p 'fizz' }
+    p 'buzz'
```

Comment:
Note that you must print `Fizz` & `Buzz`! Your current output gives double quotes on every single line. The correct output is in `correct_output.dat` (merge from my repo to yours).